src/main/java/server/db/SongDB.java

```diff
@@ -0,0 +1,163 @@
+package server.db;
+
+import org.mindrot.jbcrypt.BCrypt;
+import server.models.Song;
+import server.models.User;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+public class SongDB {
+    public static final List<Song> songs = new ArrayList<>();
+
+    private static Connection mConn;
+
+    static {
+        try {
+            Class.forName("org.postgresql.Driver");
+            String url = "jdbc:postgresql://localhost:5432/spotifylite";
+
+            try {
+                mConn = DriverManager.getConnection(url);
+                ResultSet results = mConn.createStatement().executeQuery("SELECT * FROM songs");
+                while (results.next()) {
+                    int id = results.getInt("songid");
+                    String username = results.getString("username");
+                    String artist = results.getString("artist");
+                    String title = results.getString("title");
+                    String path = results.getString("path");
+                }
+            } catch (SQLException e) {
+                e.printStackTrace();
+                System.out.println("DB error.");
+            }
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
+            System.out.println("Postgres driver not configured correctly.");
+        }
+    }
+
+    public static Song createSong(String username, String artist, String title, String path) {
+        String sql = "INSERT INTO songs(username, artist, title, path) VALUES('%s', '%s', '%s', '%s') RETURNING (songid);";

```

Comment:
Not sure if you learned about SQL Injection, but one thing to be aware of is that constructing any SQL statement like this can allow SQL injection attacks. There's a thing called prepared statements that can be used to prevent this.